cumulusci/utils.py

```diff
@@ -146,11 +146,15 @@ def zip_inject_namespace(zip_src, namespace=None, managed=None, filename_token=N
     zip_dest = zipfile.ZipFile(io.BytesIO(), 'w', zipfile.ZIP_DEFLATED)
 
     for name in zip_src.namelist():
-        content = zip_src.read(name).replace(namespace_token, namespace)
-        content = content.replace(namespaced_org_token, namespaced_org)
-        name = name.replace(filename_token, namespace)
-        name = name.replace(namespaced_org_file_token, namespaced_org)
-        zip_dest.writestr(name, content)
+        try:
+            content = zip_src.read(name).replace(namespace_token, namespace)
```

Comment:
Could we limit the number of statements in the try block to just what is needed? In other words, which statement(s) do we actually expect to raise this? Seems it would only be the `zip_src.read(name)`.